zulip_bots/zulip_bots/bots/susi/susi.py

```diff
@@ -39,11 +39,39 @@ def handle_message(self, message: Dict[str, str], bot_handler: Any) -> None:
         if msg == 'help' or msg == '':
             bot_handler.send_reply(message, self.usage())
             return
-        reply = requests.get("https://api.susi.ai/susi/chat.json", params=dict(q=msg))
+        reply = requests.get('https://api.susi.ai/susi/chat.json', params=dict(q=msg))
         try:
-            answer = reply.json()['answers'][0]['actions'][0]['expression']
-        except Exception:
-            answer = "I don't understand. Can you rephrase?"
+            data = reply.json()
+            actions = data['answers'][0]['actions']
+            answer = ''
+            for action in actions:
+                if action['type'] == 'answer':
+                    answer += action['expression'] + '\n'
+                elif action['type'] == 'anchor':
+                    text = action['text']
+                    link = action['link']
+                    answer += '[' + text + '](' + link+')' + '\n'
```

Comment:
missing space before binary operator here.

Reply:
minor, but I think this should be fixed eventually as well.

Reply:
@tonythomas01 fixed. thanks for pointing out. :)